src/bindings/python/tests/test_ngraph/test_basic.py

```diff
@@ -74,6 +76,17 @@ def test_ngraph_function_api():
         np.uint16,
         np.uint32,
         np.uint64,
+        Type.f16,
+        Type.f32,
+        Type.f64,
+        Type.i8,
+        Type.i16,
+        Type.i32,
+        Type.i64,
+        Type.u8,
+        Type.u16,
+        Type.u32,
+        Type.u64,
```

Comment:
Can we support `u4`/``i4` and other custom types?

Reply:
Refer to ticket **57071**.